packages/legend-application-studio/src/stores/sidebar-state/ProjectOverviewState.ts

```diff
@@ -183,6 +197,78 @@ export class ProjectOverviewState {
     }
   }
 
+  *fetchDependants(): GeneratorFn<void> {
+    const groupId =
+      this.editorStore.projectConfigurationEditorState.projectConfiguration
+        ?.groupId;
+    const artifactId =
+      this.editorStore.projectConfigurationEditorState.projectConfiguration
+        ?.artifactId;
+    const version = this.editorStore.sdlcState.projectVersions[0]?.id.id;
+
+    if (!groupId || !artifactId || !version) {
+      return;
+    }
+
+    try {
+      this.projectDependantEditorState.fetchingDependantInfoState.inProgress();
+      const data =
+        yield this.editorStore.depotServerClient.getIndexedDependantProjects(
```

Comment:
could we follow what we do in other places and actually seralize the `getIndexedDependantProjects` result into the expected class.